### PR TITLE
added overwrite flag to clone

### PR DESCRIPTION
**Pull request by rohitsanj**
With `--overwrite` flag, you can now do:

`jovian clone user/projectname --overwrite` and it will overwrite the folder `projectname`. 

and fixed `include_outputs` CLI option.

### CODE REVIEW

File: jovian/utils/clone.py [hunk 1]
```diff
@@ -97,8 +98,10 @@ def clone(slug, version=None, fresh=True, include_outputs=True):
             set_notebook_slug(f['filename'], slug)
 
     # Print success message and instructions
-    if fresh:
+    if fresh and not overwrite:
         post_clone_msg(title)
+    elif overwrite:
+        log(f'Downloaded files into {title}')
```
Comment (rohitsanj) [inline jovian/utils/clone.py:104]: This message can be improved.. Any suggestions? @aakashns @siddhantujjain

File: jovian/utils/clone.py [hunk 2]
```diff
@@ -77,10 +77,11 @@ def clone(slug, version=None, fresh=True, include_outputs=True):
     # If fresh clone, create directory
     if fresh:
         if os.path.exists(title):
-            i = 1
-            while os.path.exists(title + '-' + str(i)):
-                i += 1
-            title = title + '-' + str(i)
+            if not overwrite:
```
Comment (aakashns) [inline jovian/utils/clone.py:80]: nested `if`s often lead to bugs and make it harder to understand the code. Can we refactor this to remove nesting?

There are 4 cases here:
- fresh & doesn't already exist (overwrite doesn't matter): 
- fresh & exists & overwrite
- fresh & exists & doesn't overwrite
- not fresh (existence & overwrite don't matter)

Would be best to flatten out to these 4 conditions, and refactor any repeated code into helper functions.

File: jovian/utils/clone.py (hunk 1 above)
Comment (aakashns) [reply jovian/utils/clone.py:104]: No need to change the message here. `post_clone_msg` should do

File: jovian/utils/clone.py (hunk 2 above)
Comment (rohitsanj) [reply jovian/utils/clone.py]: Done, please review again :)

File: jovian/utils/clone.py [hunk 3]
```diff
@@ -75,12 +75,18 @@ def clone(slug, version=None, fresh=True, include_outputs=True):
...
+
+    elif fresh and os.path.exists(title) and overwrite:
+        os.chdir(title)
+
+    elif fresh and os.path.exists(title) and not overwrite:
+        i = 1
+        while os.path.exists(title + '-' + str(i)):
+            i += 1
+        title = title + '-' + str(i)
         if not os.path.exists(title):
```
Comment (aakashns) [inline jovian/utils/clone.py:90]: this `if` can be removed

Comment (rohitsanj) [reply jovian/utils/clone.py]: removed

**Pull request merged**